website/docs/litmusctl/litmusctl-syntax.md

```diff
@@ -0,0 +1,364 @@
+- **Custom Location:** Use `--config` flag to specify a different location
+- **No Merging:** If `--config` is specified, only that file is loaded (no merging with default config)
+
+---
+
+## Common Workflows
+
+### Initial Setup
+```bash
+# 1. Configure your account
+litmusctl config set-account
+
+# 2. View configured accounts
+litmusctl config get-accounts
+
+# 3. Create or select a project
+litmusctl create project --name="my-project"
+litmusctl get projects
+```
+
+### Connecting Infrastructure
+```bash
+# 1. Create an environment (if needed)
+litmusctl create chaos-environment
+
+# 2. Connect chaos infrastructure
+litmusctl connect chaos-infra
+
+# 3. Verify connection
+litmusctl get chaos-infra
+```
+
+### Running Experiments
+```bash
+# 1. Create experiment
+litmusctl create chaos-experiment -f experiment.yml --project-id="<id>" --chaos-infra-id="<id>"
+
+# 2. Run experiment
+litmusctl run chaos-experiment
+
+# 3. List experiments
+litmusctl get chaos-experiments
+```
+
+---
+
+## Examples
+
+### Example 1: Complete Setup in Non-Interactive Mode
+```bash
+# Set account
+litmusctl config set-account \
+  --endpoint="https://chaos.example.com" \
+  --username="admin" \
+  --password="password"
+
+# Create project
+litmusctl create project --name="production-chaos"
+
+# Get project ID
+litmusctl get projects
+
+# Connect infrastructure
+litmusctl connect chaos-infra \
+  --project-id="50addd40-8767-448c-a91a-5071543a2d8e" \
+  --environment-id="prod-env" \
+  --chaos-infra-name="prod-cluster-1" \
+  --installation-mode="cluster" \
+  --platform-name="GKE" \
+  --namespace="litmus" \
+  --non-interactive
+```
+
+### Example 2: Interactive Workflow
+```bash
+# Configure account interactively
+litmusctl config set-account
+# Follow prompts for endpoint, username, password
+
+# Connect infrastructure interactively
+litmusctl connect chaos-infra
+# Follow prompts for all required details
+
+# Create and run experiment interactively
+litmusctl create chaos-experiment -f my-experiment.yml
+# Follow prompts for project-id and chaos-infra-id
+
+litmusctl run chaos-experiment
+# Follow prompts for project-id and experiment-id
+```
+
+---
+
+## Tips and Best Practices
+
+1. **Use Descriptive Names:** Give meaningful names to your infrastructure and experiments for easier management
+2. **Leverage Non-Interactive Mode:** Use non-interactive mode in CI/CD pipelines and automation scripts
+3. **Version Compatibility:** Always check compatibility between litmusctl version and ChaosCenter version
+4. **Multiple Accounts:** Use `config get-accounts` and `config use-account` to manage multiple ChaosCenter environments
+5. **Namespace Mode vs Cluster Mode:** Choose installation mode based on your RBAC requirements and blast radius preferences
+
+---
+
+## Additional Resources
+
+- For detailed usage examples, refer to the [litmusctl usage documentation](https://github.com/litmuschaos/litmusctl/blob/master/Usage_0.23.0.md)
```

Comment:
Link https://docs.litmuschaos.io/docs/next/litmusctl/litmusctl-usage instead here

Please check other docs for ref (to understand how to link)